pkgs/development/libraries/czmqpp/default.nix

```diff
@@ -0,0 +1,34 @@
+{ stdenv, fetchurl, autoconf, automake, libtool, pkgconfig, czmq }:
+
+stdenv.mkDerivation rec {
+  name = "czmqpp-${version}";
+  version = "1.2.0";
+
+  src = fetchurl {
+    url = "https://github.com/zeromq/czmqpp/archive/v${version}.tar.gz";
+    sha256 = "0aiyv0vpzi0d9gd5ap77zjkn6mkzi1bq71n26pqb1id7rz1q7naf";
+  };
+
+  meta = with stdenv.lib; {
+    homepage = "https://github.com/zeromq/czmqpp";
+    description = "C++ wrapper for czmq. Aims to be minimal, simple and consistent.";
+    license = licenses.lgpl3;
+    maintainers = with maintainers; [ chris-martin ];
+  };
+
+  buildInputs = [ autoconf automake libtool pkgconfig ];
+
+  propagatedBuildInputs = [ czmq ];
+
+  # https://github.com/zeromq/czmqpp/issues/42
+  patches = [ ./socket.patch ];
+
+  configurePhase = ''
+    autoreconf -i
+    ./configure --prefix=$out
+  '';
```

Comment:
I'm pretty sure pkgconfig is added somewhere automatically but I'm on my mobile and can't verify right now